### PR TITLE
[FEMDEM] Correcting minor things

### PR DESCRIPTION
**Pull request by AlejandroCornejo**

### CODE REVIEW

File: applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py [hunk 1]
```diff
@@ -136,6 +143,8 @@ def SolveSolutionStep(self):
 
 #============================================================================================================================
 	def GenerateDEM(self): # 3D version
+		if self.echo_level > 0:
+			KratosMultiphysics.Logger.PrintInfo("FEM-DEM:: GenerateDEM")
```
Comment (loumalouomega) [inline applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py:147]: I don't know of there is a conditional KratosInfo on python, I would approve, but check it please

File: applications/MeshingApplication/custom_utilities/mmg_utilities.cpp [hunk 2]
```diff
@@ -1,3128 +0,0 @@
-// KRATOS  __  __ _____ ____  _   _ ___ _   _  ____
```
Comment (loumalouomega) [inline applications/MeshingApplication/custom_utilities/mmg_utilities.cpp]: DO NOT REMOVE THIS!!!!!!!

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Revert remove of applications/MeshingApplication/custom_utilities/mmg_utilities.cpp

File: applications/MeshingApplication/custom_utilities/mmg_utilities.cpp (hunk 2 above)
Comment (AlejandroCornejo) [reply applications/MeshingApplication/custom_utilities/mmg_utilities.cpp]: sorry, was not intended

File: applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py (hunk 1 above)
Comment (AlejandroCornejo) [reply applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py:147]: I've serached in Kratos and at least no one is using it.. :S

Comment (AlejandroCornejo) [reply applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py:147]: I've searched in Kratos and no one is using it at least... :S

**Review by loumalouomega — APPROVED** [review]
OK now bro

File: applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py (hunk 1 above)
Comment (philbucher) [reply applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py:147]: I don't think this exists in python

**Pull request merged**